src/core/adapters/gitlab.js

```diff
@@ -6,14 +6,27 @@
 
 import { $has, $text } from './helpers';
 
+function findTicketId(doc) {
+  const ticketId = doc.body.dataset.pageTypeId;
+
+  if (ticketId) {
+    return ticketId;
+  }
+
+  const initialDataEl = doc.getElementById('js-issuable-app-initial-data');
+
+  // Legacy approach of extracting the ticket id, left in place to support
+  // older self-hosted GitLab installations
```

Comment:
```suggestion
  // Legacy approach of extracting the ticket id, left in place to support
  // older self-hosted GitLab installations
  const initialDataEl = doc.getElementById('js-issuable-app-initial-data');
```
Would place this after the comment.